Recipes/Recipes/Supporting Files/StateShimmer.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  StateShimmer.swift
+//  Recipes
+//
+//  Created by Vitaliy Iakushev on 06.03.2024.
```

Comment:
```suggestion
//  Created by Vitaliy Iakushev on 06.03.2024.
```
поправьте форматер все шапки кривые